### PR TITLE
function to find the next empty cell

### PR DESCRIPTION
**Pull request by kruti-w**
Part of the solver algorithm

### CODE REVIEW

File: src/app/components/board.jsx [hunk 1]
```diff
@@ -32,3 +32,18 @@ export default function Board() {
...
   )
 }
+
+function nextEmptyCell(cells, index){
+  for (let i = index; i < 81; i++){
+    if (cells[i] == ''){ // or do we write null???
+      return i;
+    }
+  }
+  for (let i = 0; i < index + 1; i++){
```
Comment (raswonders) [inline src/app/components/board.jsx:42]: This seems to have off by 1 error.

**Review by raswonders — CHANGES_REQUESTED** [review]
I'm not exactly sure how this function falls into the algo you're implementing I need to see more context I guess.

File: src/app/components/board.jsx (hunk 1 above)
Comment (kruti-w) [reply src/app/components/board.jsx]: Fixed it.

**Comment by kruti-w** [comment]
https://github.com/kruti-w/tangerine-moose-sudoku/tree/solver-algorithm - Solver implementation begun here. I haven't made a PR yet but please feel free to jump in on it! I'll be back online in about 5 hours and can collab then

**Comment by raswonders** [comment]
I think this is already outdated PR as solve algo is implemented. let me know if we can close it.

**Pull request closed without merging**